repos/fdbt-site/src/pages/api/selectSalesOfferPackage.ts

```diff
@@ -83,7 +87,28 @@ export const sanitiseReqBody = (
         errors,
     };
 };
-
+export const getProductsByValues = (
+    ticket: TicketWithIds | undefined,
+    multipleProductAttribute: MultipleProductAttribute | MultipleProductAttributeWithErrors | undefined,
+    pricePerDistanceName: string,
+): ProductInfo[] => {
+    // for 'period', 'multiOperator', 'flatFare' that have multiple products
+    const products = [{ productName: 'product', productPrice: '' }];
```

Comment:
could we remove this line and replace the final return statement to `return [{ productName: 'product', productPrice: '' }]`